zebra-network/src/config.rs

```diff
@@ -745,6 +761,29 @@ impl<'de> Deserialize<'de> for Config {
                     params_builder = params_builder.with_network_magic(Magic(network_magic));
                 }
 
+                if let Some(slow_start_interval) = slow_start_interval {
+                    params_builder = params_builder.with_slow_start_interval(
+                        slow_start_interval.try_into().map_err(de::Error::custom)?,
+                    );
+                }
+
+                if let Some(target_difficulty_limit) = target_difficulty_limit {
+                    params_builder = params_builder.with_target_difficulty_limit(U256::from(
+                        <[u8; 32]>::try_from(
+                            hex::decode(target_difficulty_limit).map_err(de::Error::custom)?,
+                        )
+                        .map_err(|err| {
+                            de::Error::custom(format!(
+                                "hex-encoded target difficulty must be 64 chars: {err:?}"
+                            ))
+                        })?,
+                    ));
```

Comment:
This should work too if we want to simplify this:

```suggestion
                    params_builder = params_builder.with_target_difficulty_limit(
                        target_difficulty_limit
                            .parse::<U256>()
                            .map_err(de::Error::custom)?,
                    );
```